hydromt_wflow/wflow.py

```diff
@@ -2626,9 +2630,9 @@ def setup_lulcmaps_with_paddy(
           reduced (Feddes) [cm]
         * **h4** map: Soil water pressure head h4 at which root water uptake is reduced
           (Feddes) [cm]
-        * **h_min** map: Minimum required water depth for paddy fields [mm]
-        * **h_opt** map: Optimal water depth for paddy fields [mm]
-        * **h_max** map: Maximum water depth for paddy fields [mm]
+        * **demand_paddy_h_min** map: Minimum required water depth for paddy fields [mm]
+        * **demand_paddy_h_opt** map: Optimal water depth for paddy fields [mm]
+        * **demand_paddy_h_max** map: Maximum water depth for paddy fields [mm]
```

Comment:
Maybe for these, as they are defined in the config better to use the wflow variable name and value?

```suggestion
        * **land~irrigated-paddy__min_depth** value: Minimum required water depth for paddy fields [mm]
        * **land~irrigated-paddy__optimal_depth** value: Optimal water depth for paddy fields [mm]
        * **land~irrigated-paddy__max_depth** value: Maximum water depth for paddy fields [mm]
```